lib/jsdom/browser/Window.js

```diff
@@ -556,6 +556,97 @@ function Window(options) {
 
     forEachSheetRuleOfElement(elt, setPropertiesFromRule);
 
+    /**
+     * properties for which `getResolvedValue` is implemented which is less than
+     * every supported property according to step 5
+     * @type Record<string, { inherited: boolean; initial: unknown; computedValue: 'as-specified' }>
+     */
+    const properties = {
```

Comment:
I think this could move to the top of the file, which would be cleaner IMO, especially if the goal is to expand it.

Reply:
Actually, maybe it'd be best to move this, and most (all?) of the functions here, into a separate file? E.g. jsdom/lib/jsdom/living/helpers/style-rules.js

Reply:
done